2023/10/25 00:57:59 failed to dial tcp connection, retrying
[2023-10-25T00:58:00.190Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-10-25T00:58:00.191Z] info  Using user-data-dir ~/.local/share/code-server
[2023-10-25T00:58:00.205Z] info  Using config file ~/.config/code-server/config.yaml
[2023-10-25T00:58:00.205Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-10-25T00:58:00.205Z] info    - Authentication is disabled
[2023-10-25T00:58:00.205Z] info    - Not serving HTTPS
====================================>token: ecc08554-f02f-4207-be38-87bde416eb64
[00:58:01] 




[00:58:01] Extension host agent started.
[00:58:03] [127.0.0.1][c033d0db][ManagementConnection] New connection established.
[00:58:03] [127.0.0.1][e78d4754][ExtensionHostConnection] New connection established.
[00:58:04] [127.0.0.1][e78d4754][ExtensionHostConnection] <379> Launched Extension Host Process.
[00:58:08] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[00:58:19] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-10-25T01:00:12.856Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8986f1]

goroutine 314 [running]:
http_reverse_proxy/client.sendHttpRequestAndReplyHttpResponse(0xc9c9fc77c4986711?, 0xf874182fe3322f00?, 0xc0000d0060, 0xc0000de0c0)
	/builds/hitminer/http_reverse_proxy/client/http_client.go:51 +0xb1
created by http_reverse_proxy/client.startHttpClient
	/builds/hitminer/http_reverse_proxy/client/http_client.go:34 +0x1ea
